core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -11,6 +12,30 @@ class ReligionManager {
 
     var pantheonBelief: String? = null
     
+    var greatProphetsEarned = 0
+
+    fun clone(): ReligionManager {
+        val clone = ReligionManager()
+        clone.pantheonBelief = pantheonBelief
+        clone.storedFaith = storedFaith
+        return clone
+    }
+
+    fun startTurn() {
+        if (canGenerateProphet() && storedFaith > faithForNextGreatProphet()) {
```

Comment:
Surely the storedFaith > ... should go *inside* canGenerateProphet? Or call it 'shoudGenerateProphet'